tests/application/test_authorization.py

```diff
@@ -450,8 +460,16 @@ def test_finish_authorization_decrease(
     with ape.reverts():
         taco_application.finishAuthorizationDecrease(staking_provider, sender=creator)
 
+    chain.pending_timestamp += deauthorization_duration // 4
+    remaining_duration = deauthorization_duration - deauthorization_duration // 4
+    assert (
+        taco_application.remainingAuthorizationDecreaseDelay(staking_provider)
+        == remaining_duration - 1
```

Comment:
Is it `-1` because `current` is 1 less than `pending`?

Reply:
it should opposite, current timestamp - next block after pending. To be honest that part is 100% on test chain and sometimes hard to predict in advance do we need to subtract second or not. At least it's stable